addons/account/data/mail_template_data.xml

```diff
@@ -44,6 +44,10 @@
                 <br /><br />
                 Please use the following communication for your payment: <span style="font-weight:bold;" t-out="object.payment_reference or ''">INV/2021/05/0005</span>.
             </t>
+        </t>
+         <t t-if="hasattr(object, 'timesheet_ids') and object.timesheet_ids">
```

Comment:
We could use `timesheet_count` instead since we just want to know if there is at least one timesheet linked to that invoice.
```suggestion
         <t t-if="hasattr(object, 'timesheet_count') and object.timesheet_count">
```

Reply:
Hello @xavierbol 
I have done necessary changes
Thank You :)